src/main/java/com/diamondfire/helpbot/bot/command/impl/stats/individualized/CpCommand.java

```diff
@@ -92,7 +92,11 @@ protected void execute(CommandEvent event, String player) {
                             .compile()
                             .run((tableSet) -> {
                                 if (!tableSet.isEmpty()) {
-                                    embed.addField("Highest Point Count", FormatUtil.formatNumber(tableSet.getResult().getInt("points")), false);
+                                    if (Integer.getInteger(genPointMetric(points, uuid)) > Integer.getInteger(FormatUtil.formatNumber(tableSet.getResult().getInt("points")))) {
```

Comment:
Integer.getInteger() will not work with these methods. I recommend adding a variable names ``points`` that stores tableSet.getResult().getInt("points") and then ``topPoint`` which stores getPointMetric(). You then can compare these numbers instead of using Integer.getInteger()